plugins/importexport/datacite/filter/DataciteXmlFilter.inc.php

```diff
@@ -164,15 +164,20 @@ function &process(&$pubObject) {
 		// Publication Year (mandatory)
 		$rootNode->appendChild($node = $doc->createElementNS($deployment->getNamespace(), 'publicationYear', date('Y', strtotime($publicationDate))));
 		// Subjects
-		$subject = null;
+		$subjects = null;
```

Comment:
No need to change this, but just as a tip I'd usually prefer to make this `$subjects = [];`. That way, we won't have mixed types. It will always be an array. Just a tiny bit more consistency.

Reply:
@NateWr, this is a good point! I wanted to change it too but... :-) So I think I will change it and then merge and port...
Thanks a lot!